charts/alertmanager/templates/statefulset.yaml

```diff
@@ -228,6 +228,8 @@ spec:
   volumeClaimTemplates:
     - metadata:
         name: storage
+        labels:
+          {{- include "alertmanager.labels" . | nindent 10 }}
```

Comment:
Setting dynamic labels in `statefulset.spec.volumeClaimTemplates` like `app.kubernetes.io/version` will prevent an upgrade unless the statefulset gets deleted. This is because those fields are immutable and cannot be patched. Labels in such a case have to be fairly static, e.g. corresponding to the selector labels. It is useful to define a set of labels specifically for statefulset's persistent volume claims if such labels are desired.